src/constants.js

```diff
@@ -65,3 +65,5 @@ export const HATHOR_TOKEN_CONFIG = {'name': 'Hathor', 'symbol': 'HTR', 'uid': '0
  * Hathor token default index
  */
 export const HATHOR_TOKEN_INDEX = 0;
+
+export const GTM_ID = 'GTM-MJVX6BG';
```

Comment:
Will we be sending both testnet and mainnet events using the same GTM_ID?
Shouldn't they have one ID each?

Reply:
Changed to env var

Reply:
Maybe we could have only mainnet tag